webapp/components/create_post.jsx

```diff
@@ -77,7 +82,10 @@ export default class CreatePost extends React.Component {
             fullWidthTextBox: PreferenceStore.get(Preferences.CATEGORY_DISPLAY_SETTINGS, Preferences.CHANNEL_DISPLAY_MODE, Preferences.CHANNEL_DISPLAY_MODE_DEFAULT) === Preferences.CHANNEL_DISPLAY_MODE_FULL_SCREEN,
             showTutorialTip: false,
             showPostDeletedModal: false,
-            enableSendButton: false
+            enableSendButton: false,
+            lastBlurAt: 0,
```

Comment:
I see `lastBlurAt` being added to the state here, but it's not used anywhere. Is that an accident? If so can we remove it?